internal/rest/avi_obj_vsvip.go

```diff
@@ -497,3 +496,18 @@ func (rest *RestOperations) AviVsVipCacheDel(rest_op *utils.RestOp, vsKey avicac
 
 	return nil
 }
+
+func networkNamesToVips(networkNames []string) []*avimodels.Vip {
+	var vipList []*avimodels.Vip
+	autoAllocate := true
+	for vipIDInt, networkName := range networkNames {
+		vipID := strconv.Itoa(vipIDInt + 2)
```

Comment:
Just a query: Any reason you had to add 2 here ?

Reply:
vip ID starts from index 1 and not 0.
earlier argument was that first vip will be created using networkname.
updated it to  "+ 1" instead accordingly